### ISSUE TITLE
[Suggestion] Support for Custom Item Tiers?

### ISSUE THREAD

**Issue by KiichigoUsagi** [Judgy53/ConfigEgocentrism#6]
There could/should be support for Custom item tiers like the Void Lunar items from BubbetsItems unless there already is support for that and it uses specific internal tier labels.

**Comment by Judgy53** [comment]
Agreed. Currently, tiers are hard-coded, and unknown tiers are ignored. Not entirely sure how to make it user friendly (how does the end user know what the internal name of the modded tier is), but definitely a feature I would like in this mod.
